### DISCUSSION TITLE
Possible visual misinterpretation of JSON

### DISCUSSION THREAD

**Discussion by gurubaladhinesh** [AykutSarac/jsoncrack.com#170] [General]
When there are multiple key nodes at same level in JSON, keeping the key whose value is an Array alone in a separate box gives a wrong meaning OR does not give the meaning that all key nodes are at same level

Reg. the default JSON example provided in 'https://jsoncrack.com/editor'

Though 'members' key is at same level(in JSON) with 'squadName', the visual representation of having 'members' in a separate box and an arrow leading from 'squadName' box to 'members' box might give a wrong impression about the level of nodes.

<img width="1280" alt="Screenshot 2022-09-13 at 4 33 13 PM" src="https://user-images.githubusercontent.com/17271333/189885641-898030a9-6ef1-4f89-9748-7526234ef911.png">

Solution for the above misinterpretation is open for discussion.

**Comment by AykutSarac** [comment]
That's the expected behaviour actually, it means "members" is part of the previous node and a parent (either an array or object).
